clkhash/rest_client.py

```diff
@@ -24,6 +28,11 @@ class RateLimitedClient(ServiceError):
     """
 
 
+def retry_if_reate_limited_error(exception):
```

Comment:
```suggestion
def _is_rate_limited_error(exception):
```

Type in *reate* and (despite the retry docs) it is more pythonic to use `is_...`